README.md

```diff
@@ -31,6 +94,37 @@ Severities are used to examine and filter server logs, create alerts, and analyz
 
 *Invalid Use* - inappropriate inputs from a user - are not immediately actionable for a development team, but may provide input into customer care, future features, or bug fixes for poorly implemented features. This makes them a discrete category similar to `INFO` so that any queries on the logs can be readily consumed by such information seekers.
 
+### How it Works
+
+Logatron, when deployed in a non-rails environment, acts as a simple logger.
+When deployed in a rails environment, the `logatron/railtie` is loaded. This
+railtie uses the gem lograge under the hood. It is lograge that is used to
+write the one-line rails HTTP request log line. Logatron works to do all the
+configuration necessary of both lograge and payload methods off the base
+controller class.
+
+Perhaps the most non-intuitive operation the logatron railtie does is perform
+an "around alias" on the `.append_info_to_payload` method in the base
+controller specified in the logatron configuration (see 
+`lib/logatron/railtie.rb`). This method is part of the 
+`ActiveSupport::Notifications` code mixed in the `ActionController` in rails by
+default. Lograge depends upon this notification code as a trigger to write the
+rails HTTP request log line. 
+
+The around-aliasing of the `.append_info_to_payload` was done so that consumers
+of logatron do not need to concern themselves with specifying that method
+themselves. You are welcome.
+
+## Testing
+
+There is one test worthy of note, `spec/lib/logatron/railtie_spec.rb`. This
+test launches a real rails application via the gem combustion. Any necessary
+files in addition to a typical rails application are specified in the folder
+`spec/internal`. It is here you can see the configuration of logatron
+specified in the `_log.rb` file. The downside of this testing is that rails
+can be initialized only one time, so only one rspec scenario is testing the
+railtie.
```

Comment:
Is the 'only one rspec scenario' portion of this still true? Does the fix for making it work with the other specs also fix the multi-spec issue?

Reply:
The fix did not address being able to load rails multiple times. That is still a constraint.